.editorconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2018 Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
```

Comment:
Have we decided that `insert_final_newline` should be false? I thought we needed it. 
Either way, not a big deal.

Reply:
No official resolution yet